### ISSUE TITLE
Yellow text in Vagrantfile hard to read

### ISSUE THREAD

**Issue by paulschreiber** [Varying-Vagrant-Vagrants/VVV#1303]
## Current Behavior
The yellow text generated when you start up a VVV VM is very hard to read against a white background.

![screenshot 2017-10-09 14 02 15](https://user-images.githubusercontent.com/86784/31351826-88c6618c-acfa-11e7-94a4-608b6987dacf.png)

For reference, #FFDE13 : #ffffff has a contrast ratio of 1.34:1. WCAG suggest a ratio of 4.5:1.

## Possible Solution
Change `38;5;220` to another color.

## Steps to Reproduce (for bugs)
vagrant ssh

## Your Environment
* VVV version: 2.0.0
* Vagrant version: 1.9.7
* VM Provider name: Virtualbox
* VM Provider version: 5.1.26
* Operating System and version: macOS 10.12.6

**Comment by grappler** [comment]
I agree that checking the contrast is important. Most people can decide themselves what background colour they would like to use. This would make it impossible to make the text readable for every colour.

For example it looks fine for me on a black background.
![image](https://user-images.githubusercontent.com/1785641/31369882-209c367c-ad90-11e7-9705-b920a690711c.png)

I am not sure what the solution is. Do we define certain background colours that we support or do we allow users to define their own colours easily?

**Comment by paulschreiber** [comment]
Since the default Terminal background color is white, the text should have enough contrast to be read in that state.

**Comment by tomjn** [comment]
I'm open to suggestions and pull requests

**Comment by paulschreiber** [comment]
Do you have an ANSI color chart you use? I spent 15 minutes looking in to this, and had a hard time making sense of the escape sequences.

**Comment by tomjn** [comment]
best way to test is to put a shell script in a VVV folder and `vagrant ssh` and execute it, I usually google every time to find a colour chart, it's all very sub-optimal =/

The other thing to keep in mind is that you then need to escape the strings in ruby else the vagrantfile will be invalid

**Comment by tomjn** [comment]
I'm happy for the yellow to be replaced by colour resets though, then it'd be black on white terminals and white on black ones

**Comment by paulschreiber** [comment]
A reset seems like a good way to fix this.

**Comment by tomjn** [comment]
Thoughts?

<img width="486" alt="screen shot 2017-10-22 at 16 59 03" src="https://user-images.githubusercontent.com/58855/31863806-f7de5b74-b74a-11e7-9bda-aa077d512580.png">
<img width="543" alt="screen shot 2017-10-22 at 16 59 24" src="https://user-images.githubusercontent.com/58855/31863807-f7f89f70-b74a-11e7-89fc-2a6985e66678.png">

**Comment by paulschreiber** [comment]
That looks great!
